test/parallel/test-buffer-alloc.js

```diff
@@ -1027,7 +1027,8 @@ assert(Buffer.allocUnsafe(1).parent instanceof ArrayBuffer);
 Buffer.poolSize = ps;
 
 // Test Buffer.copy() segfault
-assert.throws(() => Buffer.allocUnsafe(10).copy());
+assert.throws(() => Buffer.allocUnsafe(10).copy(),
+            /TypeError: argument should be a Buffer/);
```

Comment:
Can you line this up with the beginning of the arrow function on the previous line. Also, if you're validating the entire error message, you can add `^` and `$` to the regular expression.